.vscode/launch.json

```diff
@@ -51,10 +51,12 @@
       "type": "node",
       "request": "attach",
       "port": 9229,
-      "protocol": "inspector",
       "timeout": 600000,
       "preLaunchTask": "Debug: Excel Desktop",
-      "postDebugTask": "Stop Debug"
+      "postDebugTask": "Stop Debug",
+      "resolveSourceMapLocations": [
+        "${env:LOCALAPPDATA}/Microsoft/Office/16.0/Wef/**/*.js"
```

Comment:
So this tells VS Code debugger to pay attention to source maps that are in the WEF cache directory, right?

Reply:
Yes, it will look to any source files there